src/PJ_isea.c

```diff
@@ -1039,6 +1038,12 @@ static XY s_forward (LP lp, PJ *P) {           /* Spheroidal, forward */
     struct isea_pt out;
     struct isea_geo in;
 
+    if (pj_is_nan(lp.lam) || pj_is_nan(lp.phi)) {
+        proj_log_error(P, "isea lp input has a NaN: lam: %lf phi: %lf",
+                       lp.lam, lp.phi);
+        return xy;
```

Comment:
Should I be calling this?
```C
proj_errno_set(P, PJD_ERR_INVALID_X_OR_Y);
```

Reply:
Definitely better to do the NaN check in pj_fwd.  I very much doubt that any project is expecting a NaN.

Reply:
I disagree...

I can plot a coastline in MATLAB using NaNs to cause the breaks which
are necessary to skip from one island to the next.  Reprojecting the
data should silently preserve NaNs.

Reply:
Yes, this is fine.

However, as a matter of programming policy, I prefer to avoid special
handling for NaNs -- just let them percolate through the code.  This
keeps the code simple.  NaNs are (should be) rare so there's not much
benefit to an early exit from a function.

Unfortunately this policy hits a glitch with the double to int
conversion (which is why I wish that the C standard said that the
*result* of the conversion of NaN to int is undefined rather than that
the *behavior* is undefined).

Finally, note that the treatment of NaNs requires attention to
conditionals.  E.g., the convergence criterion should not be
`abs(d) <= eps` but `!(abs(d) > eps)`.

Reply:
> Yes, this is fine.

Great. Question is then, @schwehr, is that enough to avoid the problem your fuzzer uncovered? From you description the input seem to be actual numbers.